src/main/java/org/opentripplanner/routing/core/vehicle_sharing/VehiclePricingPackage.java

```diff
@@ -0,0 +1,271 @@
+package org.opentripplanner.routing.core.vehicle_sharing;
+
+import java.math.BigDecimal;
+
+public class VehiclePricingPackage {
+
+    private BigDecimal packagePrice;
+
+    private int packageTimeLimitInSeconds;
+
+    private int freeSeconds;
+
+    private BigDecimal minRentingPrice;
```

Comment:
Czy BigDecimal jest tu potrzebne?

Reply:
Technicznie rzecz biorąc racja, że nie jest to bezwzględnie potrzebne. Zdecydowałam się na to bardziej "dla porządku". Wykorzystuję tą wartość podczas zwracania pojazdu w taki sposób, że porównuję całkowitą cenę z minimalną i zwracam albo jedną, albo drugą wartość. Całkowitą cenę zdecydowałam się przechowywać w BigDecimal, bo przez błędy związane z zaokrągleniami wychodziły naprawdę duże różnice w cenie. To z kolei powoduje, że i tak w którymś momencie musiałabym tą wartość zamienić na BigDecimal...